lumicks/pylake/fitting/models.py

```diff
@@ -273,19 +273,21 @@ def dsdna(name, kilobase_pairs, kilobase_pairs_per_micron=0.34, temperature=24.5
     ----------
     name : str
         Name for the model. This name will be prefixed to the model parameter names.
-    kilobase_pairs : integer
-        The number of kilobase-pairs in the sample
-    kilobase_pairs_per_micron: float
-        The number of kilobase-pairs evaluating to 1 micron. This is used to convert the kilobase-pairs to microns for the fit function.
+    dna_length_kbp: integer
+        The length of the dna in sample measured in kilobase-pairs
+    um_per_bp: float
+        The number of base pairs evaluating to 1 um. This is used to convert the length in kbp to um as applied in the fit function.
     temperature: float
-        The temperature in celsius. This is used to calculate the boltzmann * temperature value
+        The temperature in celsius. This is used to calculate the boltzmann * temperature (kT) value
     """
     import scipy.constants as spy
 
     model = odijk(name)
-    model.defaults[f"{name}/Lc"].value = kilobase_pairs * kilobase_pairs_per_micron
-    # Based on lambda dna C-trap experiments
-    model.defaults[f"{name}/Lp"].value = 56.0
+    model.defaults[f"{name}/Lc"].value = dna_length_kbp * um_per_bp
+    # Nucleic Acids: Structures, Properties, and Function. By Victor A. Bloomfield, Donald M. Crothers, and Ignacio Tinoco, Jr. University Science Books: Sausalito, CA; 2000. Clothbound, 800 pp, $88.00, ISBN 0-935702-49-0.
+    model.defaults[f"{name}/Lp"].value = 50.0
+    # https://www.pnas.org/doi/pdf/10.1073/pnas.1407197111
```

Comment:
Same as before. Should go in the docstring. We use APA style:
`Lipfert, J., Skinner, G. M., Keegstra, J. M., Hensgens, T., Jager, T., Dulin, D., ... & Dekker, N. H. (2014). Double-stranded RNA under force and torque: similarities to and striking differences from double-stranded DNA. Proceedings of the National Academy of Sciences, 111(43), 15408-15413.`